package.json

```diff
@@ -1,16 +1,9 @@
 {
-  "name": "@editorjs/editorjs",
+  "name": "@discolearn/editorjs",
   "version": "2.27.0-rc.1",
-  "description": "Editor.js — Native JS, based on API and Open Source",
+  "description": "Disco patch of Editor.js — Native JS, based on API and Open Source",
```

Comment:
I think you could just have the description say `Fork of Editor.js - Native ...` because people should understand that a form means it is patched.

The extra version makes sense as well.